### PR TITLE
add profilePage

### PR DESCRIPTION
**Pull request by Kevinwey**
Pls check :)

### CODE REVIEW

File: pages/ProfilePage/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <option value="beginner">Beginner</option>
+          <option value="intermediate" selected>
+            Intermediate
+          </option>
+          <option value="advanced">Advanced</option>
+        </select>
+      </form>
+    </div>
+  );
+}
```
Comment (AnjaSiewert) [inline pages/ProfilePage/index.js:45]: - use a component, e.g. Form to render al the Form Data on the profile page

File: pages/ProfilePage/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <label aria-label="weight" htmlFor="weight">
+          Weight:
+        </label>
+        <input id="weight" type="text" name="weight" value="84"></input>
+
+        <label aria-label="heigth" htmlFor="heigth">
+          Heigth:
+        </label>
+        <input id="heigth" type="text" name="heigth" value="171"></input>
+
```
Comment (AnjaSiewert) [inline pages/ProfilePage/index.js:22]: typo: it is spelled height, not heigth :)

**Review by AnjaSiewert — CHANGES_REQUESTED** [review]
LGTM, good job! Just two issues, see my comments

File: pages/ProfilePage/index.js (hunk 1 above)
Comment (Kevinwey) [reply pages/ProfilePage/index.js:45]: Thats part of my next US

File: pages/ProfilePage/index.js (hunk 2 above)
Comment (Kevinwey) [reply pages/ProfilePage/index.js:22]: I´ll fix that

**Pull request merged**